bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html

```diff
@@ -0,0 +1,413 @@
+
+      <div class="head-intro">
+        <p>
+        {% trans %}
+          The more voices, perspective, languages, and people contributing to the
+          Web, the richer the experience for everyone. But the whole Internet is
+          not yet accessible, welcoming, and safe for all. Together, we have the
+          power to shape the Web, and our world along with it.
+        {% endtrans %}
+          <strong>
+        {% trans %}
+        A healthy Internet is open to you.
+        {% endtrans %}
+          </strong>
+        </p>
+      </div>
+    </div>
+
+    <nav class="section head-pagenav">
+      <div class="content">
+        <h2 class="head-pagenav-title">
+        {% trans %}
+          Here are three areas where we can make a difference.
+        {% endtrans %}
+        </h2>
+
+        <ul>
+          <li><a class="nav-one" href="#diversity">{{ _('Promote Diversity') }}</a></li>
+          <li><a class="nav-two" href="#respect">{{ _('Practice Respect') }}</a></li>
+          <li><a class="nav-three" href="#access">{{ _('Support Universal Access') }}</a></li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <section id="diversity" class="section topic topic-first">
+
+    <section class="topic-intro">
+      <div class="content">
+
+        <div class="topic-intro-main">
+          <h2 class="topic-title"><span>{{ _('Digital Inclusion means promoting diversity') }}</span></h2>
+
+          <p class="topic-tagline">
+          {% trans %}
+            For the Internet to fulfill its greatest promise, it must reflect the
+            diversity and experience of all people, everywhere.
+          {% endtrans %}
+          </p>
+
+          <p>
+          {% trans %}
+            As inclusive as the Web can seem, it’s not yet an equal playing field.
+            More than half the world is still without it; emerging economies and
+            marginalized communities are often the last to gain access. Far fewer
+            women are using the Internet than men. And without diversity among its
+            creators, the Web itself will reflect unconscious biases, while
+            personalizing algorithms can reinforce our own.
+          {% endtrans %}
+          </p>
+        </div>
+
+        <aside class="topic-intro-aside">
+          <div class="note">
+            <p>
+            {% trans %}
+              In nine developing countries, women are still nearly 50% less likely to
+              access the Internet than men.
+            {% endtrans %}
+            </p>
+
+            <p class="note-source">
+              <cite>
+              {% trans wwwf='http://webfoundation.org/about/research/womens-rights-online-2015/' %}
+                Source: <a href="{{ wwwf }}">World Wide Web Foundation</a>, 2015
+              {% endtrans %}
+              </cite>
+            </p>
+          </div>
+          <span class="aside-accent" aria-hidden="true" role="presentation">50%</span>
```

Comment:
It certainly could, but then it needs an ID or unique class for each box so they can have different content in the CSS. Trying to plan ahead for more of these pages, this seemed a bit DRYer and the lesser evil.

I also thought there may be the need for these to be localizable in some cases, depending on the content. Right now they're all numbers so we don't need to stringify them (probably?), though I'm now having second thoughts about "1B" on the privacy page. Maybe they do need to be strings after all. I'm not sure about "x%" in other languages now that I think about it.

Alright, I've convinced myself. I'll stringify them.